README.md

```diff
@@ -1 +1,52 @@
-# Sweetsour Parser
+<p align="center">
+  <img alt="Sweetsour" src="/docs/logo.png?raw=true" width="200px" />
+</p>
+<h1 align="center">Sweetsour</h1>
+<p align="center">
+  A CSS-in-JS parser and pipeline adhering to the ISTF spec 🍭
```

Comment:
I think there is an ambiguity here, because it parses CSS Strings with JS interpolations which seems to be a subset of cssinjs, but reading it one may not understand why cssinjs needs parsing

Reply:
What about this:

"A CSS parser with JavaScript interpolations support, compiling to ISTF"

Reply:
alternatively "A template string CSS parser"

Reply:
“Template String CSS parser”; I Love it 😍